src/sentry/static/sentry/app/views/settings/settingsIndex.tsx

```diff
@@ -324,11 +325,8 @@ const HomeLinkIcon = styled(HomeLink)`
 `;
 
 const ExternalHomeLink = styled(
-  ({
-    isCentered,
-    ...props
-  }: CenterableProps & React.ComponentProps<typeof ExternalLink>) => (
-    <ExternalLink {...props} />
+  (props: CenterableProps & React.ComponentProps<typeof ExternalLink>) => (
+    <ExternalLink {...omit(props, 'isCentered')} />
```

Comment:
Did this needed to be changed? On one hand it's a bit cleaner, on the other it pulls in lodash/omit.